linkerd/app/inbound/src/lib.rs

```diff
@@ -267,10 +276,10 @@ impl transport::metrics::TransportLabels<Endpoint> for TransportLabels {
     }
 }
 
-impl transport::metrics::TransportLabels<Source> for TransportLabels {
+impl transport::metrics::TransportLabels<ServerProtocol> for TransportLabels {
     type Labels = transport::labels::Key;
 
-    fn transport_labels(&self, source: &Source) -> Self::Labels {
-        transport::labels::Key::accept("inbound", source.tls_peer.as_ref())
+    fn transport_labels(&self, proto: &ServerProtocol) -> Self::Labels {
+        transport::labels::Key::accept("inbound", proto.tls.peer_identity.as_ref())
```

Comment:
Note: this will change to encode additional metadata, including detected protocol and SNI names